share/picongpu/dockerfiles/ubuntu-1604/start_foil_8.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -l
+#
+
+isaac &
+server_id=$!
+
+echo ""
+#echo "Let's watch a laser-plasma movie!"
+#echo "  http://laser.plasma.ninja/ngc/interface.htm"
+echo "Let's create some openPMD HDF5 files from a novel"
+echo "plasma ion accelerator driven by a short, intense"
+echo "laser pulse!"
+echo ""
+
+# wait until server is up
+sleep 5
+
+# start PIConGPU
+cd /opt/picInputs/foil
+tbg \
+  -f \
+  -s "bash -l" \
+  -c etc/picongpu/8.cfg \
+  -t etc/picongpu/bash/mpirun.tpl \
+  /tmp/foil8_001
```

Comment:
possible improvement: add an optional parameter to the scripts to control the output path